### PR TITLE
jQuery

### PR DESCRIPTION
**Pull request by sindresorhus**
I've created a quick example on how to use vanilla jQuery to create the todo app.

### CODE REVIEW

File: todo-example/jquery/js/app.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+*/
+jQuery(function($){
+
+	var Utils = {
+		// https://gist.github.com/823878
+		uuid: function() {
+			var uuid = "", i, random;
+			for ( i = 0; i < 32; i++ ) {
+				random = Math.random() * 16 | 0;
+				if ( i == 8 || i == 12 || i == 16 || i == 20 ) {
```
Comment (urrgur) [inline todo-example/jquery/js/app.js]: triple equal!

**Comment by urrgur** [comment]
You didn't run jshint or jscs .

**Comment by samccone** [comment]
@urrgur Feel free to open a pr  fixing this 😁

**Comment by sindresorhus** [comment]
Lol, that was fixed years ago. JSCS didn't even exist then. @urrgur look at the date before commenting ;)

**Pull request merged**